packages/styles/stories/Components/shellbar/shellbar.stories.js

```diff
@@ -126,28 +125,25 @@ export const ProductMenuAndSearch = () => `<div style="height:200px">
                     </nav>
                 </div>
             </div>
-        <div class="fd-shellbar__subtitle">Subtitle</div>
+            <div class="fd-shellbar__subtitle">Subtitle</div>
         </div>
-        <div class="fd-shellbar__group fd-shellbar__group--actions">
-            <div class="fd-shellbar__action fd-shellbar__action--desktop">
+        <div class="fd-shellbar__group fd-shellbar__group--center">
+            <div class="fd-shellbar__action">
                 <div class="fd-popover__control">
                     <div aria-label="Image label" onclick="onPopoverClick('F4GcX348b')" aria-controls="F4GcX348b" aria-expanded="false" aria-haspopup="true">
-                        <div class="fd-input-group fd-shellbar__input-group">
-                            <input aria-label="search-input" type="text" class="fd-input fd-input-group__input fd-shellbar__input-group-input" id="F4GcX348b1" placeholder="Search">
-                            <span class="fd-input-group__addon fd-shellbar__input-group-addon fd-input-group__addon--button">
-                                <button aria-label="button-decline" class="fd-shellbar__button fd-button fd-button--transparent">
-                                        <i class="sap-icon--decline"></i>
+                        <div class="fd-input-group fd-shellbar__search-field">
```

Comment:
this is a breaking change, isn't it?

Reply:
@platon-rov from what I've seen in ngx, it wasnt used at all, but yes, it's a breaking change

Reply:
Please rebase and rewrite commit message to note breaking changes/